.env

```
@@ -0,0 +1 @@
+VITE_API_KEY=293805c00692398c046c66e6b7b51d4d
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)